### PR TITLE
New package: setroot-2.0.1

### PR DESCRIPTION
**Pull request by SolitudeSF**

### CODE REVIEW

File: srcpkgs/setroot/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'setroot'
+pkgname=setroot
+version=2.0.1
+revision=1
+build_style=gnu-makefile
+build_options="xinerama"
+build_options_default="xinerama"
+desc_option_xinerama="Multiple monitors support"
+CFLAGS+=" -fPIE -pie"
+LDFLAGS+=" -fPIE"
```
Comment (Vaelatern) [inline srcpkgs/setroot/template]: no. These two are wrong.
Perhaps `make_use_env=compliant`?
We have CFLAGS already. The makefile should respect them.

Comment (SolitudeSF) [reply srcpkgs/setroot/template]: i changed these lines to suggested but it reverts to throwing error ```non-PIE executable found in PIE build```

Comment (Vaelatern) [reply srcpkgs/setroot/template]: Then somewhere it isn't respecting our LDFLAGS.

**Comment by SolitudeSF** [comment]
Added ```export CFLAGS="$CFLAGS $LDFLAGS"```  to pre_build.

**Pull request closed without merging**